cirq/google/api/v2/batch.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+import "cirq/google/api/v2/program.proto";
+import "cirq/google/api/v2/result.proto";
+import "cirq/google/api/v2/run_context.proto";
+
+package cirq.google.api.v2;
+
+option java_package = "com.google.cirq.google.api.v2";
+option java_outer_classname = "BatchProto";
+option java_multiple_files = true;
+
+// A Batch of multiple circuits that should be run together
+// as one QuantumProgram within Quantum Engine.
+//
+// Note: Batching is done on a best-effort basis.
+// Circuits will be be bundled together, but the size
+// of the total batch and different hardware constraints may
+// cause the programs to be executed seperately on the hardware.
+message BatchProgram {
+
+  // The circuits that should be bundled together as one program
+  repeated Program programs = 1;
+}
+
+// A batch of contexts for running a bundled batch of programs
+// To be used in conjunction with BatchProgram
+message BatchRunContext {
+
+  // Run contexts for each program in the BatchProgram
+  // Each RunContext should map directly to a Program in the corresponding
+  // BatchProgram.
+  //
+  // This message should have one RunContext for each Program in the
```

Comment:
"should" should be "must" when describing the relationship between run contexts and programs.

Reply:
Done.